response_operations_ui/cloud/cloudfoundry.py

```diff
@@ -6,8 +6,10 @@ class ONSCloudFoundry(object):
     def __init__(self):
         self._cf_env = cfenv.AppEnv()
 
-    def __bool__(self):
-        return True if self._cf_env.app else False
+    @property
+    def detected(self):
+        return self._cf_env.app
 
-    def redis(self, service_name):
-        return self._cf_env.get_service(name=service_name)
+    @property
+    def redis(self):
+        return self._cf_env.get_service(name='ras-redis')
```

Comment:
We should probably be able to set the redis service name with an environment variable